### DISCUSSION TITLE
Importing results from Multiphysics Software for mechanical loading simulation

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#2495] [Q&A]
Hello everyone,

I need to create a mechanical simulation after a multiphysic one created with Comsol. Is it possible to import results from those platform in the Openradioss environment? I thought about a .csv conversion of the results but I don't have any idea for the transfer in Openradioss but manually input.

Does anyone have already carry on this kind of simulation?

**Comment by MarianBulla** [comment]
Hello @FGarelli,
what kind of results to you want to import into OpenRadioss ?

Can you provide some more details, please ?

Thanks and best regards,
Marian
